### PR TITLE
Release candidate for v3.4.2

### PR DESCRIPTION
**Pull request by james-d-mitchell**

### CODE REVIEW

File: makedoc.g [hunk 1]
```diff
@@ -6,12 +6,12 @@
...
 
-PACKAGE := "Semigroups";
 LoadPackage("GAPDoc");
 LoadPackage("semigroups");
 
 SemigroupsMakeDoc();
 CopyHTMLStyleFiles("doc");
-GAPDocManualLab(PACKAGE);
+# The next step loads the Semigroups package!
+GAPDocManualLab("Semigroups");
```
Comment (wilfwilson) [inline makedoc.g:15]: The Semigroups package is already loaded on line 10.

Comment (james-d-mitchell) [reply makedoc.g:15]: Right! But it would be possible to build the doc without loading the package, if this line and line 10 weren't included

Comment (james-d-mitchell) [reply makedoc.g:15]: I should remove this comment

Comment (wilfwilson) [reply makedoc.g:14]: Ah I see 🙂

**Pull request closed without merging**